include/openssl/evp_mac.h

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
About this file name, wouldn't `mac.h` be enough?  Other headers have paved the way already, such as `kdf.h`, and I really don't see a need to have `evp_` in the header file name.

Reply:
Can do.